server/channels/jobs/mobile_versions/worker.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package mobile_versions
+
+import (
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+	"github.com/mattermost/mattermost/server/v8/channels/jobs"
+	"github.com/mattermost/mattermost/server/v8/channels/store"
+	"github.com/mattermost/mattermost/server/v8/einterfaces"
+)
+
+func MakeWorker(jobServer *jobs.JobServer, store store.Store, getMetrics func() einterfaces.MetricsInterface) *jobs.SimpleWorker {
+	const workerName = "MobileVersions"
+
+	isEnabled := func(cfg *model.Config) bool {
+		return *cfg.MetricsSettings.EnableClientMetrics
+	}
+	execute := func(logger mlog.LoggerIFace, job *model.Job) error {
+		defer jobServer.HandleJobPanic(logger, job)
+
+		metrics := getMetrics()
+		if metrics == nil {
+			return nil
+		}
+
+		versions, err := store.Session().GetMobileVersions()
```

Comment:
How about the other prop we started collecting as well for disabled notifications on device? Could we include that as well?

Reply:
I should be doable. I will add it.